src/runtime/config/configuration-clh.toml.in

```diff
@@ -105,6 +105,12 @@ default_memory = @DEFMEMSZ@
 # This is will determine the times that memory will be hotadded to sandbox/VM.
 #memory_slots = @DEFMEMSLOTS@
 
+# Default maximum memory per SB / VM
```

Comment:
Suggest a change to `Default maximum memory in MiB per SB / VM`, add memory size unit will be helpful like `default_memory`.